server/config/mq-config.js

```diff
@@ -8,6 +8,20 @@ const mqSchema = joi.object({
     reconnect_Limit: joi.number().default(10),
     transport: joi.string().default('tcp')
   },
+  sqsCalculationQueue: {
+    url: joi.string().default(''),
```

Comment:
The queue configs should fail schema validation if the uri, accessKeyId or secretAccessKey are not populated so a misconfigured service fails immediately and provides details of what settings are missing.